scripts/download.pl

```diff
@@ -85,16 +87,42 @@ sub tool_present {
 	return $present
 }
 
+sub select_tool {
+	$custom_tool =~ tr/"//d;
+	if ($custom_tool) {
+		return $custom_tool;
+	}
```

Comment:
just a question, the config is a string with only `""` when it's empty, so in perl it's not a null string? and removing the characters makes it a null string?

anyway, I think this regex should be done when custom_tool is first defined, if it even needs to be done at all. I don't know Perl very well but it apparently ignores double quotes for the most part, compared with a shell where double quotes decides how much it is expanded

Reply:
well lost too much times discovering that... ENV variable are with "" so i have to strip them to be detected as NULL ahahha

Reply:
not all variables in ENV, but just the configs that are strings right?

Reply:
yes as they will be set but set to `""` as empty